app/models/story.model.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, Sequelize) => {
+    const Story = sequelize.define("story", {
+        userId: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        story: {
+            type: Sequelize.STRING(5000),
```

Comment:
Possibly just make this?
```suggestion
            type: Sequelize.TEXT,
```
So we don't have to hard-code numbered limits.